server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -85,12 +85,12 @@ public int estimatedNumberOfOperations() {
         return numberOfOperations;
     }
 
-    /** the size of the generations in the translog that weren't yet to comitted to lucene */
+    /** the size of the generations in the translog that weren't yet to committe to lucene */
```

Comment:
`committed`

Reply:
Thanks. I've fixed this and the same one in the same file as well.